app/scripts/views/tooltip.js

```diff
@@ -97,13 +97,13 @@ define([
       this._destroy = _.bind(this.destroy, this, true);
       // keyboard input for input/select elements.
       invalidEl.one('change', this._destroy);
-      // destroy the tooltip if a key different than tab is pressed
+      // destroy the tooltip if other key than tab or arrow is pressed
       var destroyIfNotTab = function (e) {
-        if (e.which !== 9) {
+        if (e.which !== 9 && ! (e.which > 36 && e.which < 41)) {
```

Comment:
@npestana this looks like a reasonable approach, though I think this can be cleaned up slightly. Can you create a named constant for each key value, and then list each explicitly?

e.g.:
```js
// up top with the other constants
var DOWN_ARROW_CODE = <code>;
var ENTER_CODE = 13;
var LEFT_ARROW_CODE = <code>;
var TAB_KEY_CODE = 9;
var RIGHT_ARROW_CODE = <code>;
var UP_ARROW_CODE = <code>;

var keysThatDoNotCloseTooltip = [
  DOWN_ARROW_CODE,
  LEFT_ARROW_CODE,
  TAB_KEY_CODE,
  RIGHT_ARROW_CODE,
  UP_ARROW_CODE
];

function doesKeyCloseTooltip(keyCode) {
  return keysThatDoNotCloseTooltip.indexOf(keyCode) === -1;
}

...
// further below in destroyIfNotTab
if (doesKeyCloseTooltip(e.which)) {
  this._destroy();
} else {
  invalidEl.one('keydown', destroyIfNotTab);
}
```

`destroyIfNotTab` should be renamed to something like `closeIfKeyNotIgnored` or `closeIfKeyClosesTooltip` so that its intent is clear and the function name no longer lies (i.e., `IfNotTab` is no longer correct after this update).

Finally, we'll need some updated unit tests [in the tooltip unit test module](https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/views/tooltip.js)